src/main/webapp/web.xml

```diff
@@ -9,6 +9,96 @@
     </servlet>
     <servlet-mapping>
         <servlet-name>index</servlet-name>
-        <url-pattern>/index</url-pattern>
+        <url-pattern>/</url-pattern>
```

Comment:
where do you call your controllers? I didn't see you modifying index controller or jsp. what does your home page show?

Reply:
> home page or some kind of menu is not required in the scope of this task

fair point, but how should we test our application? =)

Reply:
@Max-Pochepets manually typing the url's in the search bar